frontend/src/pages/dashboard/profile/EditProfile.tsx

```diff
@@ -65,30 +65,51 @@ export default function EditProfileForm() {
     sncsCertificationId: Yup.number().optional(),
   });
 
-  const emptyFile = {} as File;
-
-  const defaultValues = {
-    ...currentUser,
-    avatar: {
-      ...emptyFile,
-      preview: currentUser?.profilePictureUrl || '',
-    },
-    favouriteGymIds: currentUser?.favouriteGyms?.map((gym) => gym.id),
-  };
+  // const emptyFile = useMemo(() => ({} as File), []);
+
+  const defaultValues = useMemo(
+    () => ({
+      name: currentUser.name ? currentUser.name : undefined,
+      height: currentUser.height ? currentUser.height : undefined,
+      reach: currentUser.reach ? currentUser.reach : undefined,
+      pronounId: currentUser.pronounId ? currentUser.pronounId : undefined,
+      highestBoulderingGradeId: currentUser.highestBoulderingGradeId
+        ? currentUser.highestBoulderingGradeId
+        : undefined,
+      highestTopRopeGradeId: currentUser.highestTopRopeGradeId
+        ? currentUser.highestTopRopeGradeId
+        : undefined,
+      highestLeadClimbingGradeId: currentUser.highestLeadClimbingGradeId
+        ? currentUser.highestLeadClimbingGradeId
+        : undefined,
+      favouriteGymIds: currentUser.favouriteGyms
+        ? currentUser.favouriteGyms?.map((gym) => gym.id)
+        : undefined,
+      sncsCertificationId: currentUser.sncsCertificationId
+        ? currentUser.sncsCertificationId
+        : undefined,
+      // avatar: currentUser?.profilePictureUrl
+      //   ? {
+      //       ...emptyFile,
+      //       preview: currentUser?.profilePictureUrl,
+      //     }
+      //   : undefined,
```

Comment:
Can I confirm that we are not showing a preview of the currently uploaded profile picture in the Edit Profile page? (assuming the user has a profile picture already)

Since avatar.preview has to be filled with the profilePhotoUrl in order for the preview to appear.